plugins/modules/backup_plan.py

```diff
@@ -33,7 +33,7 @@
   rules:
     description:
       - An array of BackupRule objects, each of which specifies a scheduled task that is used to back up a selection of resources.
-      - Required when O(state=present).
+      - Required when o(state=present).
```

Comment:
```suggestion
      - Required when O(state=present).
```

Reply:
@alinabuzachis  this change is causing sanity tests to fail, hence went with `o` instead of `O`. same for other suggestions
```Running sanity test "use-compat-six"
Running sanity test "validate-modules"
ERROR: Found 1 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/backup_plan_info.py:0:0: invalid-documentation-markup: Directive "O(rules.lifecycle)" contains a non-existing option "rules.lifecycle"
See documentation for help: https://docs.ansible.com/ansible-core/2.15/dev_guide/testing/sanity/validate-modules.html
Running sanity test "yamllint"
FATAL: The 1 sanity test(s) listed below (out of 46) failed. See error output above for details.
validate-modules
```

Reply:
@mandar242 The error you're seeing is because the module does not implement that option or the path to that option is wrong. In your case you could either:

-  `To reference an option for another plugin/module plugin.fqcn.name of type type, use O(plugin.fqcn.name#type:option) and O(plugin.fqcn.name#type:option=name). For modules, use type=module. The FQCN and plugin type can be ignored by the documentation renderer, turned into a link to that plugin, or even directly to the option of that plugin.` https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation If you want to refer to when that option is set in the backup_plan module. Probably something like `returned: when O(amazon.aws.backup_plan#module:rules.lifecycle) configured
`
- Use RV() to refer the returned value. In your case it will be` RV(backup_plans.backup_plan.rules.lifecycle)`.

- Use simply `when configured`.

Reply:
thanks @alinabuzachis , this helped